move envelopes in status-protocol-go

cammellos opened the pull request:
Move envelopes tracking to status-protocol-go.

Post endpoint will not track anymore envelopes, so this is a breaking change, and version is updated accordingly.

ghost commented:
<!--prchecklist--> 
### Pull Request Checklist
- [ ] Have you updated the documentation, if impacted (e.g. [docs.status.im](https://docs.status.im/docs/build_status_go.html))?

status-im-auto commented:
### Jenkins Builds
<details>
<summary>Click to see older builds (40)</summary>

| :grey_question: | Commit | :hash: | Finished (UTC) | Duration | Platform | Result |
|-|-|-|-|-|-|-|
| :heavy_check_mark: | 00dd5686 | [#1](https://ci.status.im/job/status-go/job/prs/job/linux/job/PR-1558/1/) | 2019-08-06 09:15:32 | ~42 sec | `linux` | [:package: zip](https://status-go.ams3.digitaloceanspaces.com/status-go-desktop-190806-091449-00dd56.zip) |
| :heavy_check_mark: | 00dd5686 | [#1](https://ci.status.im/job/status-go/job/prs/job/ios/job/PR-1558/1/) | 2019-08-06 09:19:26 | ~4 min | `ios` | [:package: zip](https://status-go.ams3.digitaloceanspaces.com/status-go-ios-190806-091448-00dd56.zip) |
| :heavy_check_mark: | 00dd5686 | [#1](https://ci.status.im/job/status-go/job/prs/job/android/job/PR-1558/1/) | 2019-08-06 09:21:41 | ~6 min | `android` | [:package: aar](https://status-go.ams3.digitaloceanspaces.com/status-go-android-190806-091447-00dd56.aar) |
| | | | | | | |
| :heavy_check_mark: | a55c0811 | [#2](https://ci.status.im/job/status-go/job/prs/job/linux/job/PR-1558/2/) | 2019-08-06 15:22:36 | ~40 sec | `linux` | [:package: zip](https://status-go.ams3.digitaloceanspaces.com/status-go-desktop-190806-152156-a55c08.zip) |
| :heavy_check_mark: | a55c0811 | [#2](https://ci.status.im/job/status-go/job/prs/job/ios/job/PR-1558/2/) | 2019-08-06 15:25:22 | ~3 min | `ios` | [:package: zip](https://status-go.ams3.digitaloceanspaces.com/status-go-ios-190806-152155-a55c08.zip) |
| :heavy_check_mark: | a55c0811 | [#2](https://ci.status.im/job/status-go/job/prs/job/android/job/PR-1558/2/) | 2019-08-06 15:27:20 | ~5 min | `android` | [:package: aar](https://status-go.ams3.digitaloceanspaces.com/status-go-android-190806-152154-a55c08.aar) |
| | | | | | | |
| :heavy_check_mark: | b24b2715 | [#3](https://ci.status.im/job/status-go/job/prs/job/linux/job/PR-1558/3/) | 2019-08-06 16:17:54 | ~44 sec | `linux` | [:package: zip](https://status-go.ams3.digitaloceanspaces.com/status-go-desktop-190806-161710-b24b27.zip) |
| :heavy_check_mark: | b24b2715 | [#3](https://ci.status.im/job/status-go/job/prs/job/ios/job/PR-1558/3/) | 2019-08-06 16:20:47 | ~3 min | `ios` | [:package: zip](https://status-go.ams3.digitaloceanspaces.com/status-go-ios-190806-161709-b24b27.zip) |
| :heavy_check_mark: | b24b2715 | [#3](https://ci.status.im/job/status-go/job/prs/job/android/job/PR-1558/3/) | 2019-08-06 16:22:48 | ~5 min | `android` | [:package: aar](https://status-go.ams3.digitaloceanspaces.com/status-go-android-190806-161707-b24b27.aar) |
| | | | | | | |
| :heavy_check_mark: | d49dda3f | [#4](https://ci.status.im/job/status-go/job/prs/job/linux/job/PR-1558/4/) | 2019-08-06 18:38:56 | ~33 sec | `linux` | [:package: zip](https://status-go.ams3.digitaloceanspaces.com/status-go-desktop-190806-183823-d49dda.zip) |
| :heavy_check_mark: | d49dda3f | [#4](https://ci.status.im/job/status-go/job/prs/job/ios/job/PR-1558/4/) | 2019-08-06 18:41:07 | ~2 min | `ios` | [:package: zip](https://status-go.ams3.digitaloceanspaces.com/status-go-ios-190806-183822-d49dda.zip) |
| :heavy_check_mark: | d49dda3f | [#4](https://ci.status.im/job/status-go/job/prs/job/android/job/PR-1558/4/) | 2019-08-06 18:43:57 | ~5 min | `android` | [:package: aar](https://status-go.ams3.digitaloceanspaces.com/status-go-android-190806-183820-d49dda.aar) |
| | | | | | | |
| :heavy_check_mark: | d53d789f | [#5](https://ci.status.im/job/status-go/job/prs/job/linux/job/PR-1558/5/) | 2019-08-06 18:44:17 | ~31 sec | `linux` | [:package: zip](https://status-go.ams3.digitaloceanspaces.com/status-go-desktop-190806-184345-d53d78.zip) |
| :heavy_check_mark: | d53d789f | [#5](https://ci.status.im/job/status-go/job/prs/job/ios/job/PR-1558/5/) | 2019-08-06 18:47:37 | ~3 min | `ios` | [:package: zip](https://status-go.ams3.digitaloceanspaces.com/status-go-ios-190806-184344-d53d78.zip) |
| :heavy_check_mark: | d53d789f | [#5](https://ci.status.im/job/status-go/job/prs/job/android/job/PR-1558/5/) | 2019-08-06 18:49:37 | ~5 min | `android` | [:package: aar](https://status-go.ams3.digitaloceanspaces.com/status-go-android-190806-184359-d53d78.aar) |
| | | | | | | |
| :heavy_check_mark: | 66352209 | [#6](https://ci.status.im/job/status-go/job/prs/job/linux/job/PR-1558/6/) | 2019-08-07 09:26:09 | ~32 sec | `linux` | [:package: zip](https://status-go.ams3.digitaloceanspaces.com/status-go-desktop-190807-092537-663522.zip) |
| :heavy_check_mark: | 66352209 | [#6](https://ci.status.im/job/status-go/job/prs/job/ios/job/PR-1558/6/) | 2019-08-07 09:28:43 | ~3 min | `ios` | [:package: zip](https://status-go.ams3.digitaloceanspaces.com/status-go-ios-190807-092536-663522.zip) |
| :heavy_check_mark: | 66352209 | [#6](https://ci.status.im/job/status-go/job/prs/job/android/job/PR-1558/6/) | 2019-08-07 09:31:12 | ~5 min | `android` | [:package: aar](https://status-go.ams3.digitaloceanspaces.com/status-go-android-190807-092534-663522.aar) |
| | | | | | | |
| :heavy_check_mark: | 4b4f75bd | [#7](https://ci.status.im/job/status-go/job/prs/job/linux/job/PR-1558/7/) | 2019-08-07 12:32:54 | ~37 sec | `linux` | [:package: zip](https://status-go.ams3.digitaloceanspaces.com/status-go-desktop-190807-123216-4b4f75.zip) |
| :heavy_check_mark: | 4b4f75bd | [#7](https://ci.status.im/job/status-go/job/prs/job/ios/job/PR-1558/7/) | 2019-08-07 12:35:25 | ~3 min | `ios` | [:package: zip](https://status-go.ams3.digitaloceanspaces.com/status-go-ios-190807-123215-4b4f75.zip) |
| :heavy_check_mark: | 4b4f75bd | [#7](https://ci.status.im/job/status-go/job/prs/job/android/job/PR-1558/7/) | 2019-08-07 12:37:52 | ~5 min | `android` | [:package: aar](https://status-go.ams3.digitaloceanspaces.com/status-go-android-190807-123214-4b4f75.aar) |
| | | | | | | |
| :heavy_check_mark: | ba1a4134 | [#8](https://ci.status.im/job/status-go/job/prs/job/linux/job/PR-1558/8/) | 2019-08-07 13:58:10 | ~31 sec | `linux` | [:package: zip](https://status-go.ams3.digitaloceanspaces.com/status-go-desktop-190807-135738-ba1a41.zip) |
| :heavy_check_mark: | ba1a4134 | [#8](https://ci.status.im/job/status-go/job/prs/job/ios/job/PR-1558/8/) | 2019-08-07 14:02:16 | ~4 min | `ios` | [:package: zip](https://status-go.ams3.digitaloceanspaces.com/status-go-ios-190807-135737-ba1a41.zip) |
| :heavy_check_mark: | ba1a4134 | [#8](https://ci.status.im/job/status-go/job/prs/job/android/job/PR-1558/8/) | 2019-08-07 14:03:26 | ~5 min | `android` | [:package: aar](https://status-go.ams3.digitaloceanspaces.com/status-go-android-190807-135736-ba1a41.aar) |
| | | | | | | |
| :heavy_check_mark: | b67a0aa0 | [#9](https://ci.status.im/job/status-go/job/prs/job/linux/job/PR-1558/9/) | 2019-08-07 14:50:34 | ~38 sec | `linux` | [:package: zip](https://status-go.ams3.digitaloceanspaces.com/status-go-desktop-190807-144956-b67a0a.zip) |
| :heavy_check_mark: | b67a0aa0 | [#9](https://ci.status.im/job/status-go/job/prs/job/ios/job/PR-1558/9/) | 2019-08-07 14:52:59 | ~3 min | `ios` | [:package: zip](https://status-go.ams3.digitaloceanspaces.com/status-go-ios-190807-144954-b67a0a.zip) |
| :heavy_check_mark: | b67a0aa0 | [#9](https://ci.status.im/job/status-go/job/prs/job/android/job/PR-1558/9/) | 2019-08-07 14:55:33 | ~5 min | `android` | [:package: aar](https://status-go.ams3.digitaloceanspaces.com/status-go-android-190807-144953-b67a0a.aar) |
| | | | | | | |
| :heavy_check_mark: | 7ee91378 | [#10](https://ci.status.im/job/status-go/job/prs/job/linux/job/PR-1558/10/) | 2019-08-09 07:05:41 | ~1 min | `linux` | [:package: zip](https://status-go.ams3.digitaloceanspaces.com/status-go-desktop-190809-070431-7ee913.zip) |
| :heavy_check_mark: | 7ee91378 | [#10](https://ci.status.im/job/status-go/job/prs/job/ios/job/PR-1558/10/) | 2019-08-09 07:09:25 | ~4 min | `ios` | [:package: zip](https://status-go.ams3.digitaloceanspaces.com/status-go-ios-190809-070428-7ee913.zip) |
| :heavy_check_mark: | 7ee91378 | [#10](https://ci.status.im/job/status-go/job/prs/job/android/job/PR-1558/10/) | 2019-08-09 07:10:11 | ~5 min | `android` | [:package: aar](https://status-go.ams3.digitaloceanspaces.com/status-go-android-190809-070427-7ee913.aar) |
| | | | | | | |
| :heavy_check_mark: | 272a52b9 | [#11](https://ci.status.im/job/status-go/job/prs/job/linux/job/PR-1558/11/) | 2019-08-09 07:07:28 | ~36 sec | `linux` | [:package: zip](https://status-go.ams3.digitaloceanspaces.com/status-go-desktop-190809-070651-272a52.zip) |
| | | | | | | |
| :heavy_check_mark: | 3a0fe9d4 | [#12](https://ci.status.im/job/status-go/job/prs/job/linux/job/PR-1558/12/) | 2019-08-09 07:09:06 | ~33 sec | `linux` | [:package: zip](https://status-go.ams3.digitaloceanspaces.com/status-go-desktop-190809-070833-3a0fe9.zip) |
| :heavy_check_mark: | 3a0fe9d4 | [#11](https://ci.status.im/job/status-go/job/prs/job/ios/job/PR-1558/11/) | 2019-08-09 07:13:50 | ~4 min | `ios` | [:package: zip](https://status-go.ams3.digitaloceanspaces.com/status-go-ios-190809-070926-3a0fe9.zip) |
| :heavy_check_mark: | 3a0fe9d4 | [#11](https://ci.status.im/job/status-go/job/prs/job/android/job/PR-1558/11/) | 2019-08-09 07:15:45 | ~5 min | `android` | [:package: aar](https://status-go.ams3.digitaloceanspaces.com/status-go-android-190809-071013-3a0fe9.aar) |
| | | | | | | |
| :heavy_check_mark: | 3de0be70 | [#13](https://ci.status.im/job/status-go/job/prs/job/linux/job/PR-1558/13/) | 2019-08-12 08:06:02 | ~34 sec | `linux` | [:package: zip](https://status-go.ams3.digitaloceanspaces.com/status-go-desktop-190812-080527-3de0be.zip) |
| :heavy_check_mark: | 3de0be70 | [#12](https://ci.status.im/job/status-go/job/prs/job/ios/job/PR-1558/12/) | 2019-08-12 08:08:13 | ~2 min | `ios` | [:package: zip](https://status-go.ams3.digitaloceanspaces.com/status-go-ios-190812-080526-3de0be.zip) |
| :heavy_check_mark: | 3de0be70 | [#12](https://ci.status.im/job/status-go/job/prs/job/android/job/PR-1558/12/) | 2019-08-12 08:11:08 | ~5 min | `android` | [:package: aar](https://status-go.ams3.digitaloceanspaces.com/status-go-android-190812-080525-3de0be.aar) |
| | | | | | | |
| :heavy_check_mark: | a180f950 | [#14](https://ci.status.im/job/status-go/job/prs/job/linux/job/PR-1558/14/) | 2019-08-12 08:22:08 | ~44 sec | `linux` | [:package: zip](https://status-go.ams3.digitaloceanspaces.com/status-go-desktop-190812-082123-a180f9.zip) |
| :heavy_check_mark: | a180f950 | [#13](https://ci.status.im/job/status-go/job/prs/job/android/job/PR-1558/13/) | 2019-08-12 08:27:18 | ~5 min | `android` | [:package: aar](https://status-go.ams3.digitaloceanspaces.com/status-go-android-190812-082121-a180f9.aar) |
| :heavy_check_mark: | a180f950 | [#13](https://ci.status.im/job/status-go/job/prs/job/ios/job/PR-1558/13/) | 2019-08-12 08:30:13 | ~8 min | `ios` | [:package: zip](https://status-go.ams3.digitaloceanspaces.com/status-go-ios-190812-082122-a180f9.zip) |
</details>

| :grey_question: | Commit | :hash: | Finished (UTC) | Duration | Platform | Result |
|-|-|-|-|-|-|-|
| :heavy_check_mark: | 20542bcd | [#15](https://ci.status.im/job/status-go/job/prs/job/linux/job/PR-1558/15/) | 2019-08-12 08:34:59 | ~45 sec | `linux` | [:package: zip](https://status-go.ams3.digitaloceanspaces.com/status-go-desktop-190812-083412-20542b.zip) |
| :heavy_check_mark: | 20542bcd | [#14](https://ci.status.im/job/status-go/job/prs/job/ios/job/PR-1558/14/) | 2019-08-12 08:36:44 | ~2 min | `ios` | [:package: zip](https://status-go.ams3.digitaloceanspaces.com/status-go-ios-190812-083411-20542b.zip) |
| :heavy_check_mark: | 20542bcd | [#14](https://ci.status.im/job/status-go/job/prs/job/android/job/PR-1558/14/) | 2019-08-12 08:39:47 | ~5 min | `android` | [:package: aar](https://status-go.ams3.digitaloceanspaces.com/status-go-android-190812-083410-20542b.aar) |
| | | | | | | |
| :heavy_check_mark: | 7af9ca48 | [#16](https://ci.status.im/job/status-go/job/prs/job/linux/job/PR-1558/16/) | 2019-08-12 10:48:47 | ~1 min | `linux` | [:package: zip](https://status-go.ams3.digitaloceanspaces.com/status-go-desktop-190812-104741-7af9ca.zip) |
| :heavy_check_mark: | 7af9ca48 | [#15](https://ci.status.im/job/status-go/job/prs/job/ios/job/PR-1558/15/) | 2019-08-12 10:50:57 | ~3 min | `ios` | [:package: zip](https://status-go.ams3.digitaloceanspaces.com/status-go-ios-190812-104740-7af9ca.zip) |
| :heavy_check_mark: | 7af9ca48 | [#15](https://ci.status.im/job/status-go/job/prs/job/android/job/PR-1558/15/) | 2019-08-12 10:53:18 | ~5 min | `android` | [:package: aar](https://status-go.ams3.digitaloceanspaces.com/status-go-android-190812-104739-7af9ca.aar) |

adambabik committed:
add API to shhext to manage messages

cammellos committed:
Move envelopes tracking to status-protocol-go

This commit moves envelopes tracking to status-go.
Post endpoint is not going to track envelopes anymore, as that's taken
care on status-protocol-go side, so this is a breaking change, and
version is updated accordingly.
